src/ImageRunModal.jsx

```diff
@@ -205,7 +205,7 @@ export class ImageRunModal extends React.Component {
             publish: [],
             image: props.image,
             memory: 512,
-            cpuShares: 0,
+            cpuShares: 1024,
```

Comment:
Admitteldy I don't like hardcoding this default very much, but of course the UI needs to know what to set the value to once you enable CPU shares. But given that "0" means "use the internal default", not literally "zero shares", could we just continue to interpret it that way? I. e. with "0" the setting would just appear disabled?

Reply:
I don't think that 0 means that. 0 is not defined anywhere what it mean, and seems that `podman inspects` shows 0 wrongly.
If we don't want 1024, then I would rather suggest not having any value.

Reply:
Right, if it works with `null` and then just not gets passed, that'd be even better of course -- we leave it as internal podman default then.

Reply:
Ok, so I dropped the default value.
Forgot to change the commit message, but I'll update that on marging.